lib/sorbet/eraser/t/props.rb

```diff
@@ -32,8 +32,7 @@ def const(name, rules = {})
       private
 
       def create_prop(name, rules)
-        props << [name, rules]
-        props.sort_by!(&:first)
```

Comment:
I don't think we need to sort the props at all, and it makes even less sense when this is a hash.